### DISCUSSION TITLE
Question on Taylor factor

### DISCUSSION THREAD

**Discussion by Ericpan3** [mtex-toolbox/mtex#1284] [Ask Anything] [answered]
Hello,

I was trying to get the Taylor factor from my EBSD data with the following code: 

% define 10 percent strain
**eps = 0.1 * strainTensor(diag([1 -0.75 -0.25]))**
cs = crystalSymmetry('cubic')
**ori = orientation.byEuler(0,30*degree,15*degree,cs)**
% define a slip system
sS = slipSystem.fcc(cs)
% compute the Taylor factor
[M,b,W] = calcTaylor(inv(ori)*eps,sS.symmetrise)

I'm wondering what's the meaning of the value in line 1 and line 3?  It seems that the value of the Taylor factor does not relate with my EBSD file but these values. 

Thank you

**Comment by kilir** [comment]
Exactly. Have a look at the documentation, there's an example for EBSD data.
Cheers,
rüdiger

**Reply by Ericpan3** [reply]
Thank you Rudiger,

Which document are you referring to?

**Reply by kilir** [reply]
Have a look  here https://mtex-toolbox.github.io/SchmidtFactor.html, keep scrolling. 
Hope that helps,
Rüdiger

**Reply by Ericpan3** [reply]
Thank you.  I will work from there.

**Reply by Ericpan3** [reply]
Hello Rudiger,

I'm reworking the Taylor model here: https://mtex-toolbox.github.io/TaylorModel.html.
Interestingly, the code brings me the same plot (black and white Taylor factor graph) as the website, even with my EBSD raw data.
Maybe I'm still not familiar with the code, but could you please let me know how should I call my EBSD file in this case? Thank you

**Comment by Ericpan3** [comment, marked as answer]
> Have a look here https://mtex-toolbox.github.io/SchmidtFactor.html, keep scrolling. Hope that helps, Rüdiger
Hi, Rudiger,
I have fixed it. Thank you.
